com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/AbstractResourceDescriptionStrategy.java

```diff
@@ -162,17 +162,17 @@ public boolean createReferenceDescriptions(final EObject from, final URI exporte
   }
 
   /**
-   * Creates all implicit reference descriptions and adds them to the acceptor.
+   * Creates all inferred and implicit reference descriptions and adds them to the acceptor.
    *
    * @param resource
    *          resource to create implicit references for
    * @param acceptor
    *          acceptor
    */
-  public void createImplicitReferenceDescriptions(final Resource resource, final IAcceptor<IReferenceDescription> acceptor) {
-    ImplicitReferencesAdapter adapter = ImplicitReferencesAdapter.find(resource);
+  public void createInferredImplicitReferenceDescriptions(final Resource resource, final IAcceptor<IReferenceDescription> acceptor) {

```

Comment:
not sure if wording is precise here... i'd rather not include 'inferred' in the method's name, since the actual reference isn't really inferred. Besides, at some point of time we will probably get rid of "inferred!" fragment and start using "implicit!" only